### PR TITLE
Update release notes for version 1.3

### PR DESCRIPTION
**Pull request by QRemy**
Update release notes for version 1.3

Follow-up of #5470

### CODE REVIEW

**Comment by bkhelifi** [comment]
I guess that the list of contributors are OK, as well as the PR. I have just added Pei Yu (for the web infra)

**Comment by QRemy** [comment]
> I guess that the list of contributors are OK, as well as the PR. I have just added Pei Yu (for the web infra)

added

**Review by bkhelifi — APPROVED** [review]
Thank you very much, @QRemy

File: docs/release-notes/v1.3.rst [hunk 1]
```diff
@@ -187,16 +192,16 @@ This list is incomplete. Small improvements and bug fixes are not listed here.
 - [#5275] Add a notebook for EBL correction example (Atreyee Sinha)
 - [#5271] Allow extrapolation only along spatial dimension in `mask_safe_edisp` (Quentin Remy)
 - [#5270] Modify acceptance stacking behavior (Régis Terrier)
-- [#5269] Add leakage protection factor in the Timmer Konig algorithm (Claudio Galelli)- [#5258] Add write function to `FitResult` (Astro-Kirsty)
-- [#5255] Expose `FitResult` (Astro-Kirsty)
+- [#5269] Add leakage protection factor in the Timmer Konig algorithm (Claudio Galelli)- [#5258] Add write function to `FitResult` (Kirsty Feijen)
```
Comment (morcuended) [inline docs/release-notes/v1.3.rst:195]:
```suggestion
- [#5269] Add leakage protection factor in the Timmer Konig algorithm (Claudio Galelli)
- [#5258] Add write function to `FitResult` (Kirsty Feijen)
```

**Review by morcuended — DISMISSED** [review]
Thanks. Looks good. One of the lines contains two PR (indicated below)

File: docs/release-notes/v1.3.rst [hunk 2]
```diff
@@ -7,32 +7,33 @@
...
-- Fabio Acero
-- Régis Terrier
-- Lars Mohrmann
 - Vasco Schiavo
-- Laura Olivera Nieto
-- Axel Donath
+- Atreyee Sinha
+- Brigitta Sipőcz
+- Régis Terrier
+- Santiago Vila
```
[the remark this answers is not in the document]
Comment (QRemy) [reply docs/release-notes/v1.3.rst:36]: already there : see below

**Review by morcuended — APPROVED** [review]
Looks good, thanks.

**Review by registerrier — APPROVED** [review]
Thanks @QRemy !

**Pull request merged**